src/utils/errors.rs

```diff
@@ -210,123 +239,6 @@ fn parse_forbidden_error(source: WrappedResponseContent, message: String) -> Pin
     }
 }
 
-impl std::fmt::Display for PineconeError {
```

Comment:
These `Display` implementations are basically all up in the `#[error("message")]` annotations above each error in the enum due to deriving `thiserror::Error`.